### PR TITLE
remove arb and eth goerli networks

### PR DESCRIPTION
**Pull request by avichalp**
# Summary

This PR removes support for arbitrum and ethereum goerli networks. See #607 for details

- [x]  Are changes backward compatible with existing SDKs, or is there a plan to manage it correctly?
- [x]  Are changes covered by existing tests, or were new tests included?
- [x]  Are code changes optimized for future code readers, commenting on problematic areas to understand (if any)?
- [x]  Future-self question: Did you avoid unjustified/unnecessary complexity to achieve the goal?

### CODE REVIEW

File: docker/deployed/testnet/api/config.json [hunk 1]
```diff
@@ -52,25 +52,6 @@
         "ChainStackCollectFrequency": "15m"
     },
     "Chains": [
-        {
-            "Name": "Ethereum Goerli",
```
Comment (avichalp) [inline docker/deployed/testnet/api/config.json:56]: cc: @dtbuchholz

**Review by brunocalza — APPROVED** [review]

**Comment by asutula** [comment]
Will the validator still support read queries (and other types of gateway requests) for tables on these goerli networks?

**Comment by avichalp** [comment]
@asutula i don't think so. the tables will still be there in the validator's database but the gateway has this notion of "supported chains". it rejects any query to an unsupported chain

**Comment by brunocalza** [comment]
> Will the validator still support read queries (and other types of gateway requests) for tables on these goerli networks?

The query will work, actually. The check of supported chains does not happen in the `/query` call. Can't remember if that was by design or an accident. 

Either way, an additional comment: if we ever start the validator from scratch again, the data would be lost because the validator wouldn't pick the events of a not-supported chain (we'd still have them in the backups taken when the chain was supported). We don't have the concept of "network was supported between this interval, so pick those events up".

**Comment by asutula** [comment]
Thanks for the info @brunocalza.

> if we ever start the validator from scratch again

This implies that if any new validator joins the nework, they won't have the data. Seems we should consider the data gone/unavailable then. I could be smart to delete the tables from the validator state (and Studio's state), but that may be another conversation.

**Review by asutula — APPROVED** [review]

**Pull request merged**